build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/JdkInfo.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Xamarin.Android.Build.Utilities;
+
+namespace Xamarin.Android.BuildTools.PrepTasks
+{
+	public class JdkInfo : Task
+	{
+		[Required]
+		public ITaskItem Output { get; set; }
+
+		public override bool Execute ()
+		{
+			Log.LogMessage (MessageImportance.Low, $"Task {nameof (JdkInfo)}");
+			Log.LogMessage (MessageImportance.Low, $"  {nameof (Output)}: {Output}");
+
+			AndroidLogger.Error += (task, message) => Log.LogError ($"{task}: {message}");
+			AndroidLogger.Warning += (task, message) => Log.LogWarning ($"{task}: {message}");
+			AndroidLogger.Info += (task, message) => Log.LogMessage (MessageImportance.Low, $"{task}: {message}");
+			AndroidSdk.Refresh ();
```

Comment:
This task should take "optional" (non-`[Required]`) properties to specify the parameters to `AndroidSdk.Refresh()`, and then we should set those values to the "known" values we set. For example, `androidSdkPath` should be `$(AndroidSdkPath)` (which defaults to `$HOME/android-toolchain/sdk`), `androidNdkPath` should be `$(AndroidNdkPath)`, etc.